guides/doc-Planning_for_Project/topics/Introduction.adoc

```diff
@@ -100,7 +100,7 @@ The new Content View version is then promoted through life cycle environments.
 You can also create in-place updates of Content Views.
 This means creating a minor version of the Content View in its current life cycle environment without promoting it from the Library.
 For example, if you need to apply a security erratum to a Content View used in Production, you can update the Content View directly without promoting to other life cycles.
-For more information on content management see the {ContentManagementDocURL}[Content Management Guide].
+For more information on content management, see {ContentManagementDocURL}[_{ContentManagementDocTitle}_].
```

Comment:
I don't know if italic should be used on title with embedded link. I did notice not this in the docs.